File: Month-1/Week-01/day-01/dart/main.dart
import 'dart:io';

int _YEARINDAYS = 365;
void main() {
  stdout.writeln('Type your age');
  int age = int.tryParse(stdin.readLineSync()) ?? 0;

  stdout.writeln('Number of days: ${calculateNumberDaysByAge(age)}');
}

int calculateNumberDaysByAge(int age) {
  return age * _YEARINDAYS;
}

File: Month-1/Week-01/day-01/Go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

const (
	daysInYear = 365
)

func main() {
	reader := bufio.NewReader(os.Stdin)

	fmt.Print("Enter your age: ")

	input, _ := reader.ReadString('\n')

	ageResult, err := CalculateAge(input)

	if err != nil {
		fmt.Printf("Sorry, but something went wrong: %v", err)
	} else {
		fmt.Printf("You are %d days old \n", ageResult)
	}

}

func CalculateAge(input string) (int64, error) {

	trimmedInput := strings.TrimRight(input, "\r\n")
	parsedAge, err := strconv.ParseInt(trimmedInput, 10, 64)
	if err != nil {
		return parsedAge, err
	}

	ageinDays := parsedAge * daysInYear

	return ageinDays, err
}
